test/hotspot/jtreg/runtime/HiddenClasses/StressHiddenClasses.java

```diff
@@ -36,7 +36,6 @@
 import static java.lang.invoke.MethodHandles.Lookup.ClassOption.*;
 import jdk.test.lib.compiler.InMemoryJavaCompiler;
 
-// This test is based on vmTestbase/vm/mlvm/anonloader/share/StressClassLoadingTest.java
 public class StressHiddenClasses {
```

Comment:
Since `StressClassLoadingTest` is revised to test hidden class, this test is a subset of it.
I think this can be removed as JDK-8265694 suggests.

Reply:
Thanks Mandy. I will remove the test as the fix for JDK-8265694 after this change is pushed.